cloud-agent/service/server/src/main/scala/org/hyperledger/identus/agent/server/jobs/PresentBackgroundJobs.scala

```diff
@@ -96,13 +96,17 @@ object PresentBackgroundJobs extends BackgroundJobsHelper {
     for {
       _ <- ZIO.logDebug(s"Running action with records => $record")
       _ <- record match {
-        case PresentationRecord(id, _, _, _, _, _, _, _, ProposalPending, _, _, _, _, _, _, _, _, _, _, _, _) =>
+        case PresentationRecord(_, _, _, _, _, _, _, _, InvitationGenerated, _, _, _, _, _, _, _, _, _, _, _, _, _) =>
```

Comment:
I'm dreaming about somebody refactoring this _ _ _ _ _ _ _ _ _ _ _ code :)

Reply:
same
I'm have a hard time trying to merge those changes